test/package-check-success.yml

```diff
@@ -0,0 +1,13 @@
+- tasks:
+  - name: install ansible
+    yum: name=ansible state=2.1.0.0
+
+  - name: install ansible-lint
+    pip: name=ansible-lint
+    args:
+      state: 3.1.2
```

Comment:
So this is where my being a polyglot is bad, because I had thought all automation/CM tools could specify versions in that parameter and I forgot Ansible does not do that.

Funny thing is ansible-lint still flags that.